include/z64animation.h

```diff
@@ -313,4 +313,12 @@ typedef struct {
 typedef void (*SkinCallback)(struct Actor*, struct GlobalContext*, PSkinAwb*);
 typedef s32 (*SkinCallback2)(struct Actor*, struct GlobalContext*, s32, PSkinAwb*);
 
+typedef struct LegacyLimb {
+    /* 0x000 */ Gfx* dList;
+    /* 0x004 */ Vec3f trans;
+    /* 0x010 */ Vec3s rot;
+    /* 0x018 */ struct LegacyLimb* sibling;
+    /* 0x01C */ struct LegacyLimb* child;
+} LegacyLimb; // size = 0x20
```

Comment:
This could probably do with joining the rest of the limb structures higher up.

Reply:
Done!